springboot/scripts/start.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
+echo "---------Copy Maven Scripts to /app folder"
+mkdir /app/scripts
+cp /scripts/maven.sh /scripts/logging.sh /app/scripts
+chmod +x /app/scripts/maven.sh /app/scripts/logging.sh
+mkdir /app/.m2
+cp /scripts/jboss-settings.xml /app/.m2/settings.xml
 echo "---------Starting Maven Script---------"
-/home/app/deployments/maven.sh
+/app/scripts/maven.sh
 echo "---------End Maven Script---------"
 echo "---------Starting App---------"
-java -jar /home/app/deployments/app.jar
\ No newline at end of file
+java -Dkie.maven.settings.custom=/app/.m2/settings.xml -jar /app/app.jar
```

Comment:
I'm not sure if this is useful

Reply:
It is here because we need to setup Drools in HACEP to use settings in /app/.m2/settings.xml. It is a settings XML which is generated by HACEP maven scripts and can contain setup for external Maven repositories. Without parameter "kie.maven.settings.custom" HA-CEP will not use this settings xml.

For example our test in kie-cloud test suite for kjar update [1] will fail with following error if "kie.maven.settings.custom" is not specified:
java.lang.RuntimeException: Cannot find KieModule: org.kie.cloud:test-hacep-cloud-kjar:1.0.0-SNAPSHOT at org.drools.compiler.kie.builder.impl.KieServicesImpl.newKieContainer(KieServicesImpl.java:186) at org.drools.compiler.kie.builder.impl.KieServicesImpl.newKieContainer(KieServicesImpl.java:172) at org.kie.hacep.consumer.KieContainerUtils.getKieContainer(KieContainerUtils.java:36)

[1] https://github.com/kiegroup/kie-cloud-tests/blob/master/test-cloud/test-cloud-ha-cep/src/test/java/org/kie/cloud/hacep/HACEPKjarUpdateIntegrationTest.java#L77

Reply:
ok